app/src/main/res/layout/fragment_favorites_list.xml

```diff
@@ -1,12 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/favorites_background"
     android:orientation="vertical"
     tools:context=".changes.ChangeListFragment">
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/loading_spinner_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:visibility="gone">
+
+        <ProgressBar
+            android:layout_width="80dp"
```

Comment:
Let's move the `80dp` value into a dimension resource to define it once instead of 4 times:

``` xml
<dimen name="progress_spinner_size">80dp</dimen>
```

Reply:
:+1: Thanks for the change.